ironic_prometheus_exporter/parsers/manager.py

```diff
@@ -0,0 +1,26 @@
+from ironic_prometheus_exporter.parsers import ipmi
+
+
+class ParserManager(object):
```

Comment:
This class should be a function, unless you plan on having some state for it.

Reply:
I was thinking as a manager since in the future we will need a parser for Redfish metrics, so based on some information the Manager will choose the correct parser

Reply:
This sounds like some entry points, but how does a class help? What kind of state will the class maintain?

Reply:
After we have Redfish data we would be able to load the correct parser here, I didn't understand the state part.